docs/manual/java/gettingstarted/WhatsNext.md

```diff
@@ -0,0 +1,14 @@
+# What's Next?
+
+After working through "Getting started with Lagom", follow the path that make the most sense to you:
+
+* Learn how to integrate with Eclipse:
+    * [[Import an existing Maven project into Eclipse|EclipseMavenInt]] 
+    * [[Create a new Eclipse project using the Lagom Maven archetype|EclipseMavenNew]]
+    * [[Import an existing sbt build into Eclipse|EclipseSbt]]
+    
+* Learn how to intgrate with Intellij:
+    * [[Import an existing Maven project into Intellij|intellijMaven]]
+    * [[Import an existing sbt build into Intellij|IntellijSbtJava]]
+    
+* [[Learn about Lagom core concepts|CoreConcepts]]
```

Comment:
Yes, it's because of my unusual (and inconsistent) use of links in bullets. Just in case it proves to be too difficult to fix in the CSS, I've formatted the page differently.